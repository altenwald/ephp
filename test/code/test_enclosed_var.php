<?php

$name = "world";
$names = [ "mars", "venus" ];

print "hello ${name}\n";
print "hello {$name}\n";

print "hello ${names[0]}\n";
print "hello {$names[1]}\n";

