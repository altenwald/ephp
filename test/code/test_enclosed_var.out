hello world
hello world
hello mars
hello venus
